### PR TITLE
Add build instructions for Windows

### PR DESCRIPTION
**Pull request by ForNeVeR**
Part of #45.

### CODE REVIEW

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.46-iolevqly completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.46-iolevqly) (commit https://github.com/procxx/kepka/commit/cdaca2fd03 by @ForNeVeR)

**Review by berkus — DISMISSED** [review]

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.46-octyqrto completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.46-octyqrto) (commit https://github.com/procxx/kepka/commit/741fa92ef1 by @ForNeVeR)

File: README.md [hunk 1]
```diff
@@ -22,6 +24,28 @@ Provide paths to OpenAL-soft and Qt5 in CMAKE_PREFIX_PATH variable when configur
     cmake -G Ninja -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH="/usr/local/opt/qt5/;/usr/local/opt/openal-soft" ..
     ninja
 
+### Windows
+
+Install [vcpkg][] (no need to integrate, just install), [cmake][], [Qt][qt] 5.9
+or later and [Visual Studio][visual-studio] 2017 or later, and set the following
```
Comment (berkus) [inline README.md]: Could we remove unnecessary linebreaks here (1)

File: README.md [hunk 2]
```diff
@@ -22,6 +24,28 @@ Provide paths to OpenAL-soft and Qt5 in CMAKE_PREFIX_PATH variable when configur
...
+environment variables:
+
+- `QT_DIR`: directory where Qt binary distribution is installed, e.g.
+  `C:\Qt\5.9.1\msvc2017_64`
+- `VCPKG`: directory where VCPKG is installed, e.g. `C:\vcpkg`
+
+After that, execute the following `cmd` commands from Visual Studio developer
+command prompt:
```
Comment (berkus) [inline README.md]: And here (2) ^

**Review by berkus — CHANGES_REQUESTED** [review]

**Review by berkus — DISMISSED** [review]

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.46-alacxeen completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.46-alacxeen) (commit https://github.com/procxx/kepka/commit/8d205d57e4 by @ForNeVeR)

**Review by crackedmind — DISMISSED** [review]

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.47-fmodwarp completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.47-fmodwarp) (commit https://github.com/procxx/kepka/commit/9b7aaef576 by @berkus)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.47-betivnlc completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.47-betivnlc) (commit https://github.com/procxx/kepka/commit/d2943a89c2 by @ForNeVeR)

**Comment by crackedmind** [comment]
:white_check_mark: [Build kepka 1.0.48-msouckbq completed](https://ci.appveyor.com/project/procxx/tdesktop/build/1.0.48-msouckbq) (commit https://github.com/procxx/kepka/commit/2cbfefcd58 by @ForNeVeR)

**Pull request merged**